From 3a7b0090f88ac1343b47ae0222d1cb51f22222ef Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Fri, 22 Jun 2018 12:12:04 +0800
Subject: [PATCH 1980/2328] media: intel-ipu4p: imx355: Adjust frame rate
 slightly

Message for Open Source:
CTS test requires the frame rate not excceeding 30fps.
This patch slightly modify the frame rate to be just
a little below 30fps.

Message for Internal:
[Issue/Feature] Refer to open source message.
[Root Cause/Changes] Refer to open source message.

Change-Id: Ib0e32c7be0639facc854f71ed16c526f33050dfe
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx355.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/drivers/media/i2c/imx355.c b/drivers/media/i2c/imx355.c
index 9bdd65a..27b5cfe 100644
--- a/drivers/media/i2c/imx355.c
+++ b/drivers/media/i2c/imx355.c
@@ -177,7 +177,7 @@ static const struct imx355_reg mode_3268x2448_regs[] = {
 	{ 0x0342, 0x0e },
 	{ 0x0343, 0x58 },
 	{ 0x0340, 0x0a },
-	{ 0x0341, 0x36 },
+	{ 0x0341, 0x37 },
 	{ 0x0344, 0x00 },
 	{ 0x0345, 0x08 },
 	{ 0x0346, 0x00 },
@@ -226,7 +226,7 @@ static const struct imx355_reg mode_3264x2448_regs[] = {
 	{ 0x0342, 0x0e },
 	{ 0x0343, 0x58 },
 	{ 0x0340, 0x0a },
-	{ 0x0341, 0x36 },
+	{ 0x0341, 0x37 },
 	{ 0x0344, 0x00 },
 	{ 0x0345, 0x08 },
 	{ 0x0346, 0x00 },
@@ -275,7 +275,7 @@ static const struct imx355_reg mode_3280x2464_regs[] = {
 	{ 0x0342, 0x0e },
 	{ 0x0343, 0x58 },
 	{ 0x0340, 0x0a },
-	{ 0x0341, 0x36 },
+	{ 0x0341, 0x37 },
 	{ 0x0344, 0x00 },
 	{ 0x0345, 0x00 },
 	{ 0x0346, 0x00 },
@@ -883,8 +883,8 @@ static const struct imx355_mode supported_modes[] = {
 	{
 		.width = 3280,
 		.height = 2464,
-		.fll_def = 0xa36,
-		.fll_min = 0xa36,
+		.fll_def = 0xa37,
+		.fll_min = 0xa37,
 		.llp = 0xe58,
 		.reg_list = {
 			.num_of_regs = ARRAY_SIZE(mode_3280x2464_regs),
@@ -894,8 +894,8 @@ static const struct imx355_mode supported_modes[] = {
 	{
 		.width = 3268,
 		.height = 2448,
-		.fll_def = 0xa36,
-		.fll_min = 0xa36,
+		.fll_def = 0xa37,
+		.fll_min = 0xa37,
 		.llp = 0xe58,
 		.reg_list = {
 			.num_of_regs = ARRAY_SIZE(mode_3268x2448_regs),
@@ -905,8 +905,8 @@ static const struct imx355_mode supported_modes[] = {
 	{
 		.width = 3264,
 		.height = 2448,
-		.fll_def = 0xa36,
-		.fll_min = 0xa36,
+		.fll_def = 0xa37,
+		.fll_min = 0xa37,
 		.llp = 0xe58,
 		.reg_list = {
 			.num_of_regs = ARRAY_SIZE(mode_3264x2448_regs),
-- 
2.7.4

